policies/aws.route53.-.healthcheckstatus.json

```diff
@@ -0,0 +1,28 @@
+{
+  "policy": {
+    "actions": [
+      {
+        "category": 3,
+        "type": "event"
+      }
+    ],
+    "conditions": [
+      {
+        "analytic": "actual",
+        "level": 1,
+        "operator": "<",
+        "wildcard": "aws.route53.healthcheckstatus.*"
+      }
+    ],
+    "deleted": false,
+    "description": "HealthCheckStatus is higher than usual. HealthCheckStatus is the status of the health check endpoint that CloudWatch is checking. 1 indicates healthy, and 0 indicates unhealthy. You can view HealthCheckStatus only across all regions; data is not available for a selected region.",
```

Comment:
Can you view HealthCheckStatus for a health check across all regions? I don't see a metric for that, but I do see metrics like `healthcheckstatus.us-east-1`. What should I be looking for to see this across all regions?

Reply:
In the policy I track all possible regions by regex: "wildcard": "aws.route53.timetofirstbyte.*".
I took this description from Amazon website. Possibly it is related to the AWS console, as it doesn't show a dropdown menu like for TimeToFirstByte metric.
Should I change the description?